packages/WillowCreekApp/src/tabs/connect/index.js

```diff
@@ -29,7 +29,7 @@ const ConnectNavigator = createStackNavigator(
 const EnhancedConnect = withTheme(({ theme, ...props }) => ({
   ...props,
   screenProps: {
-    headerTintColor: theme.colors.text.primary,
+    headerTintColor: theme.colors.action.primary,
```

Comment:
This should happen in core as well :) We'll need to make this change in multiple places (I can't remember where the other location is off the top of my head) 

Related: https://3.basecamp.com/3926363/buckets/6947926/todos/2141435292/edit?replace=true

Reply:
I believe Neeley and I did that while back and couldn't find a better way. A peek at the `react-navigation`codebase I think confirmed that these values are derived locally to the instance of the usage.

Reply:
😢

Reply:
We talked about making a reusable HOC that you could use to set navigation options from theme values but figured that the additional ergonomics weren't worth the added complexity.

Reply:
I think I agree.